_pages/en_US/installing-boot9strap-(pichaxx).txt

```diff
@@ -38,7 +38,7 @@ This method of using Seedminer for further exploitation uses your `movable.sed`
 1. Select your `movable.sed` file
 1. Select "Build and Download"
 1. Wait for the process to complete
-1. Navigate to `Nintendo 3DS` -> `<ID0>` -> `<ID1>` -> `title` -> `00040000` -> `0017c100` -> `data` on your SD card
+1. Navigate to `Nintendo 3DS -> <ID0> -> <ID1> -> title -> 00040000 -> 0017c100` -> `data` on your SD card
```

Comment:
I don't think this section needed changing - I find it weird that everything before data is one codeblock and then data is a seperate codeblock